### ISSUE TITLE
"Facet" (Stacked) Chart Clarification

### ISSUE THREAD

**Issue by SachaG** [Devographics/Monorepo#256]
@shhaine you suggested adding a clarification for this specific view, what do you think of this phrasing?

> Note that size differences between segments are influenced by the respondent count for each corresponding group in the overall survey data.

<img width="940" alt="Screenshot 2023-08-11 at 14 08 00" src="https://github.com/Devographics/Monorepo/assets/358832/029dedb8-a538-425d-a4aa-fbe5208b7325">

**Comment by ShaineRosewel** [comment]
What about

_Note that the percentage breakdown for each age range (group) is influenced by ..._

I'm thinking segments and group might be confusing.

**Comment by SachaG** [comment]
Good idea. But the clarification has to work for any chart, so we can't mention "age" in text directly. So maybe just:

> Note that the percentage breakdown for each group is influenced by…

**Comment by SachaG** [comment]
OK how about:

> Note that the percentages shown above are influenced by how many respondents overall belong to each group.

**Comment by ShaineRosewel** [comment]
Okay with this
